Fix invalid rendering of meeting and proposal body texts

ahukkanen opened the pull request:
#### :tophat: What? Why?
Currently meeting and proposal body rendering can be broken as explained at #9763. This fixes the described issue.

#### :pushpin: Related Issues
- Fixes #9763

#### Testing
- Create a meeting from admin panel
- Add a list to the meeting description from the WYSIWYG editor
- Go to see the meeting page
- Check the page source
- See that there are no extra `<br>` tags within the meeting description list
- Repeat the same process for admin created proposal

andreslucena commented:
Code wise, everything looks 👌🏽 

Just a minor change. 

With this text: 

![Selection_307](https://user-images.githubusercontent.com/717367/191801468-5fa45415-c3ff-4b7a-84d7-ebd4908471af.png)

Before it was rendered like this (with the bug): 

![Selection_303](https://user-images.githubusercontent.com/717367/191801527-f84d2235-acba-4a98-a71e-2bf0bc8b0597.png)

With this PR: 

![Selection_304](https://user-images.githubusercontent.com/717367/191801606-5ee1f58a-6ab2-42c9-8936-96030d0019a5.png)

I think we should add some padding at the bottom (with CSS or however you prefer). Can you do that please?

andreslucena reviewed (changes requested):
I've left a [comment with a minor change](https://github.com/decidim/decidim/pull/9764#issuecomment-1255271455), can you check it out whenever you have time? Thanks

ahukkanen commented:
Displayed list styles should be now fixed with the latest commits. I also added a margin bottom to the lists when in editor because it bothers me (and some other editors judging that they add extra paragraphs) that the next paragraph is attached to the list above it.

While doing this (and merging with develop), I noticed we've broken the sub-lists when re-editing the lists. Reported that as a separate issue at #9866. It does not seem to be related to this PR.

andreslucena commented:
There are some broken related specs, can you check them out @ahukkanen 🙏🏽?
There's also another failing (from decidim-assemblies) that will be fixed if you merge with develop (related to #9863). Can you merge please?

ahukkanen commented:
Done @andreslucena

andreslucena commented on decidim-core/app/helpers/decidim/sanitize_helper.rb:
```diff
@@ -37,7 +37,7 @@ def decidim_sanitize_newsletter(html, options = {})
     end
 
     def decidim_sanitize_editor(html, options = {})
-      content_tag(:div, decidim_sanitize(html, options), class: %w(ql-editor ql-reset-decidim))
```
If we're removing `ql-reset-decidim` from the HTML, should we also remove it [from the CSS](https://github.com/decidim/decidim/blob/develop/decidim-core/app/packs/stylesheets/decidim/extras/_quill.scss?plain=1#L15)?

On the other hand, we should also update [this expect](https://github.com/decidim/decidim/blob/develop/decidim-meetings/spec/system/user_edit_meeting_spec.rb?plain=1#L108).

ahukkanen replied:
It's removed now.

andreslucena reviewed (approved):
👍🏽
